### ISSUE TITLE
Error out of the box

### ISSUE THREAD

**Issue by thenewdesign** [Rich-Harris/degit#340]
**Ran**
npx degit chromaui/intro-storybook-vue-template taskbox

**Got**
! could not fetch remote https://github.com/chromaui/intro-storybook-vue-template
! could not find commit hash for HEAD

Not finding many things to try w/ Google, anyone know what the issue is? Trying to do the Vue Storybook tutorial, this is step 1 :D

**Comment by ghost** [comment]
same i have that kinda error too

**Comment by yogaajie2** [comment]
make sure you have git installed. If you use VS Code change the terminal to Git Bash

**Comment by thenewdesign** [comment]
> make sure you have git installed. If you use VS Code change the terminal to Git Bash

Will try that and report back. I tend to use GitHub Desktop instead of the command line. Thanks.

**Comment by thenewdesign** [comment]
> make sure you have git installed. If you use VS Code change the terminal to Git Bash

So it looks like I did have Git installed already in addition to GitHub Desktop. Also for some reason trying it today in an identical manner ( just copying and pasting the commands from the tutorial ) seemed to work. Maybe there were just some issues w/ my network at the time or something, though I did try it on multiple different days and it failed identically each time. Whatever the case it's working now and I'm excited to dig in. Thanks again for the assist.
